.palantir/revapi.yml

```diff
@@ -1172,6 +1172,11 @@ acceptedBreaks:
         \ java.util.function.Consumer<T>)"
       justification: "Removing deprecated code"
```

Comment:
To help me understand the scope of this PR: we will collect the metrics for Variant subcolumns but the storing them as Variant to Avro files and retrieving from Avro files will be separate since that depends on Avro Variant read/write implementation. And also the pruning with such metrics will be separate as well. 

Is that correct?

Reply:
Yes, this PR translates Parquet metrics for shredded fields to a Variant object that is accepted by `InclusiveMetricsEvaluator`. For Avro, we would need to come up with a strategy for producing metrics. I'm not sure that it makes sense to because there are no shredded fields.

Reply:
Got it. I'm referring to write the metrics (in Variant object) into Iceberg manifest file (in Avro file), not about collecting metrics for Avro - we don't shred fields for Avro files so it doesn't make sense to to produce metrics.